### PR TITLE
chore: cleanup assemblies

### PR DESCRIPTION
**Pull request by AjimenezDCL**
Having scripts outside an asmdef cause some troubles when using the repo as  UPM (desktop for example). Also I took the chance to clean up some assemblies.

## Test
Load the Desktop version: https://renderer-artifacts.decentraland.org/desktop/index.html?dcl://DESKTOP-BRANCH=chore/remove-missing-script-warning, play Decentraland a little bit and then open the logs https://docs.unity3d.com/Manual/LogFiles.html.

For example in windows you'd have to open: 
`%USERPROFILE%\AppData\Local\Packages\Decentraland\Decentraland\UnityPlayer.log`
or
`%USERPROFILE%\AppData\LocalLow\Packages\Decentraland\Decentraland\Player.log`

Depending if you used the standalone version or the windows store version.

In the log search for 
>The referenced script on this Behaviour (Game Object 'Spinner') is missing!

It shouldnt appear anymore

### CODE REVIEW

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/BuilderProjectsPanel.asmdef [hunk 1]
```diff
@@ -51,7 +49,9 @@
         "GUID:ee6523961ef177343ad763fcd55c7c46",
         "GUID:f51ebe6a0ceec4240a699833d6309b23",
         "GUID:3b80b0b562b1cbc489513f09fc1b8f69",
-        "GUID:ad97ccc1a866eea46b53413c4254e6cb"
+        "GUID:ad97ccc1a866eea46b53413c4254e6cb",
+        "DeployedScenesFetcher",
+        "LandWIthAccess"
```
Comment (popuz) [inline unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/BuilderProjectsPanel.asmdef:54]: Why these 2 are without GUID?

**Review by popuz — APPROVED** [review]
As I mentioned in the comment, I am not sure that assemblies should be added by name and not by the GUID. You know better how it should be ;) 

As for testing, no Spinner missing was found in the log. The only missing components are
```
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object '<null>') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object '<null>') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object '<null>') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
The referenced script on this Behaviour (Game Object 'Input') is missing!
```

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/BuilderProjectsPanel.asmdef (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/BuilderProjectsPanel.asmdef:54]: honestly, I have no idea how that was added and why is mixing GUIDs with names (should be one or another, not both)

**Review by kwspgoliasz — APPROVED** [review]
The changes has been implemented correctly. The 'Spinner' object is no longer marked as missing in player logs.

**Pull request merged**